packages/ruleset/src/rules/index.js

```diff
@@ -1,35 +1,39 @@
+const arrayOfArrays = require('./array-of-arrays');
```

Comment:
change here involve adding the two new entries and then sorting the resulting list of requires here and exports below.